build-package.sh

```diff
@@ -430,8 +465,13 @@ while (($# > 0)); do
 		cd "$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX"
 		termux_step_post_massage
 		cd "$TERMUX_PKG_MASSAGEDIR"
-		termux_step_create_datatar
-		termux_step_create_debfile
+		if [ "$TERMUX_PACKAGE_FORMAT" = "debian" ]; then
+			termux_step_create_debian_package
+		elif [ "$TERMUX_PACKAGE_FORMAT" = "pacman" ]; then
+			termux_step_create_pacman_package
+		else
+			termux_error_exit "Unknown packaging format '$TERMUX_PACKAGE_FORMAT'."
```

Comment:
Would be nicer to have the error check in the getopt loop, right now the entire build runs and then errors in case format is unknown

Reply:
I will add error checking to options loop, but I will leave this here as fallback case.